internal/locale/locales/en-us.yaml

```diff
@@ -1748,6 +1748,11 @@ err_user_network_solution:
     If your issue persists consider reporting it on our forums at {{.V0}}.
 err_revert_refresh:
   other: Could not get revert commit to check if changes were indeed made
+err_revert_commit:
+  other: |
+    Could not revert{{.V0}} commit: {{.V1}}
+    Please ensure that the local project is synchronized with the platform and that the given commit ID belongs to the current project.
+    If this is a private project, you may need to authenticate with [ACTIONABLE]`state auth`[/RESET]
```

Comment:
Consider using AddTips. In my opinion, error output is simply a list of what was tried and/or what went wrong, and AddTips is for suggestions. This makes me wonder if we have more error messages like this.

Reply:
We do, because I initially used tips, but dropped them in favor of existing messages.

I changed those messages to use tips too.